src/OrbitGl/Button.h

```diff
@@ -12,8 +12,12 @@
 #include "Viewport.h"
 
 namespace orbit_gl {
-class Button : public CaptureViewElement {
+
+// A Button can be clicked to produce an event and also will react when the mouse is over it.
+class Button : public CaptureViewElement, public std::enable_shared_from_this<Button> {
  public:
+  const float kSymbolsPaddingSize = 3.f;
+  const float kSymbolsWide = 3.f;
```

Comment:
Nit: the same as Pierric suggested, use singular here. `kSymbolPaddingSize` and `kSymbolWidth`. Also I feel these are very specific for drawing the "+" and "-" in the button. Maybe add a comment here?

Reply:
I moved it to DrawSymbol new function without adding a comment :)